src/boot.ts

```diff
@@ -100,10 +108,22 @@ async function installBoot(
       env['JAVA_CMD'] = process.env['JAVA_CMD']
     }
 
-    await exec.exec(`./boot ${version === 'latest' ? '-u' : '-V'}`, [], {
-      cwd: path.join(destinationFolder, 'boot', 'bin'),
-      env
-    })
+    if (utils.isWindows()) {
+      await exec.exec(
+        `reg add "HKLM\\SOFTWARE\\JavaSoft\\Java Development Kit\\1.8" /v JAVAHome /d "$env:JAVA_HOME" /f`
```

Comment:
Hopefully, Java versions can be quite safely extracted from `JAVA_HOME`

Reply:
I count on that as well, but right now I tried to make it work with at least one known JDK but without success

Reply:
Most likely, this message coming from Launch4j where the minimum JRE is set to 1.7.0, so everything above that should work

Reply:
ok, it works, but for some reason, the workaround must be a part of the workflow. I'll try it this evening to check what is missing when using `exec(...)` inside of the action.